reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -323,9 +448,10 @@ private fun XSSFSheet.finalize(headerRows: Int, totalRows: Int, totalColumns: In
 
 private const val ELLIPSIS = "[...]"
 
-private fun Collection<*>.joinWithLimit(
+private fun <T> Iterable<T>.joinWithLimit(
```

Comment:
Should this maybe already be done in the first commit? In other words, is the intermediate change to `Collection<*>` meaningful?

Reply:
Yes and no :)